<%= form_with(model: user, local: true, class: "form_outer") do |f| %>
  <div class="form_inner">
    <% if user.errors.any? %>
      <div id="error_explanation">
        <h2><%= t("errors.template.header", model: user.model_name.human, count: user.errors.count) %></h2>
        <ul>
          <% user.errors.full_messages.each do |msg| %>
            <li><%= msg %></li>
          <% end %>
        </ul>
      </div>
    <% end %>
    <div class="form-group">
      <%= f.label :name %>
      <%= f.text_field :name, placeholder: "#{User.human_attribute_name :name}", class: "form-control form-title" %>
    </div>
    <div class="form-group">
      <%= f.label :email %>
      <%= f.email_field :email, placeholder: "#{User.human_attribute_name :email}", class: "form-control" %>
    </div>
    <div class="form-group">
      <%= f.label :password %>
      <%= f.password_field :password, placeholder: "#{User.human_attribute_name :password}", class: "form-control" %>
    </div>
    <div class="form-group">
      <%= f.label :password_confirmation %>
      <%= f.password_field :password_confirmation, placeholder: "#{User.human_attribute_name :password_confirmation}", class: "form-control" %>
    </div>
  </div>
  <%= f.submit class: "cp_btn re_size" %><!-- f.submit, class: "cp_btn"とするとシンタックスエラーになる -->
<% end %>
